README.md

```diff
@@ -179,8 +179,12 @@ Swagger Codegen Version    | Release Date | OpenAPI Spec compatibility | Notes
 If you're looking for the latest stable version, you can grab it directly from Maven.org (Java 8 runtime at a minimum):
 
 ```sh
+# Download current stable 2.x.x branch
 wget https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.12/swagger-codegen-cli-2.4.12.jar -O swagger-codegen-cli.jar
 
+# Download current stable 3.x.x branch
```

Comment:
It might be helpful to point out that 3.x.x is meant for OpenAPI 3.0 while 2.x.x is meant for Swagger (not sure if 3.x.x supports both Swagger and OpenAPI 3.0, though).

Reply:
Done. Great tip. Thanks Igor.